scripts/create-rule.js

```diff
@@ -199,6 +192,16 @@ function updateReadme(data) {
 	});
 	updateIndex(id);
 	updateReadme(data);
+
+	try {
+		await execa('code', [
```

Comment:
I don't think you should assume VSCode. How about using https://github.com/sindresorhus/open-editor ?

Reply:
It seems open-editor  can't start my editor on windows, I'll look into it.

Reply:
Found: vscode can't use `--goto` when opening multiple files.

Reply:
https://github.com/sindresorhus/open-editor/issues/15